src/smart_contracts/contract_vm.ml

```diff
@@ -173,11 +221,13 @@ module Make (CTX : Context.CTX) : S with module Context = CTX = struct
         let _, amount2 = R.read_amount memory addr int64_offset in
         let data = (ticket_handle, amount1, amount2) in
         let store (handle1, handle2) =
+          Format.printf "%d %d\n" (Int32.to_int handle1) (Int32.to_int handle2);
           W.write_tickets ~writer_addr [handle1; handle2] memory in
         store (Table_ops.split_ticket ctx data)
       | OWN_TICKET ->
         let _, ticket_handle = R.read_ticket memory writer_addr int32_offset in
         let store handle1 = W.write_ticket ~writer_addr handle1 memory in
+        Format.printf "handle %d\n" (Int32.to_int ticket_handle);
```

Comment:
I don't know it those printfs are intentional, but I guess using the logging library would be better, probably with `Log.debug` so we'd be able to control when that appears or not. It'd also make it more visible.

Also, if that's intentional it'd be great to have a more descriptive logging message, instead of just numbers so someone looking at the logging messages for debugging would know what's that about.

Reply:
need to remove them